tests/h/search/query_test.py

```diff
@@ -1048,7 +1048,7 @@ def test_it_limits_number_of_annotation_counts_by_user_returned(
 
 
 @pytest.fixture
-def search(pyramid_request):
+def search(pyramid_request, group_service, moderation_service):
```

Comment:
Search only depends on the group_service.

Reply:
^ This one still needs to be addressed.

Reply:
Ah, indeed. Done!